sentry-android-core/src/test/java/io/sentry/android/core/ConnectivityCheckerTest.kt

```diff
@@ -23,4 +23,9 @@ class ConnectivityCheckerTest {
     fun `isConnected won't throw exception`() {
         ConnectivityChecker.isConnected(context, mock())
     }
+
+    @Test
+    fun `getConnectionType won't throw exception`() {
+        ConnectivityChecker.getConnectionType(context, mock())
```

Comment:
Ideally the tests would assert something.
A 'does not throw' test is a last resort thing when the code isnt' testable (I guess like in this case).
Would it be possible to get it to return something and assert non null?

Reply:
I agree, but unless I mock the world, I cannot guarantee that the JVM will return something.
but definitely it's on my radar to test all the android FW related classes later on.